python/nav/maintengine.py

```diff
@@ -164,7 +164,7 @@ def create_event(subject, state, value, taskid=None):
     target = 'eventEngine'
     subsystem = 'maintenance'
     source = subsystem
-    severity = 50
+    severity = 5
```

Comment:
If what this code does, is: Setting the default severity of a maintenance event to 5, this is relevant to the user and should be documented somewhere.
Otherwise,  it is safe to ignore this comment ;)

Reply:
Experience tells me most users don't care about receiving alerts for devices going on or off maintenance (they really don't!). The fact that NAV's alert system is overloaded to store maintenance state is an uninteresting implementation detail to them.

I therefore made the judgment call to set the default severity level of maintenance events to 5 (informational). 

One could argue that the event originators should never make severity judgements (makes more sense to me, anyway), and that only event engine, as the sole generator of actual alerts in NAV, should make those calls. Then the entire event model needs to change as well, and that is another matter entirely.